src/views/RecIM/components/List/index.jsx

```diff
@@ -64,8 +88,8 @@ const ParticipantList = ({
         teamID={teamID}
         callbackFunction={(bool) => callbackFunction(bool)}
         showParticipantOptions={
-          showParticipantOptions &&
-          participant.Role !== 'Team-captain/Creator' &&
+          showParticipantOptions ||
```

Comment:
showParticipantOptions was returning false so then the buttons to promote to captain and remove from team were not showing for me. It worked when I changed to "||", however I am not sure if it would be wrong for other parts of the code.